torch_ttnn/mem_utils.py

```diff
@@ -0,0 +1,151 @@
+from torch_ttnn.passes.lowering.add_data_move_pass import is_tt_data_move, is_tt_compute
+from torch_ttnn.utils import TtnnDevice
+import ttnn
+import torch
+
+# Configs
+device_name = "wormhole"
+cores = 120
+SRAM_LIMIT = 104857600  # 100 * 1024 * 1024 bytes (100 MB)
+L1_mem = 1048576  # 1 MB
+circular_buffer = 20 * 1048576  # 20 MB
```

Comment:
It is best to have an object for this config. I will check if by any chance we already have that in TT-NN. If not, I think thats reasonable to add there.